src/tree/activitiyLog/ActivityItem.ts

```diff
@@ -3,18 +3,20 @@
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/
 
-import { AzExtParentTreeItem, AzExtTreeItem, callWithTelemetryAndErrorHandling, IActionContext, TreeItemIconPath } from "@microsoft/vscode-azext-utils";
+import { callWithTelemetryAndErrorHandling, TreeItemIconPath } from "@microsoft/vscode-azext-utils";
 import { Activity, ActivityTreeItemOptions, OnErrorActivityData, OnProgressActivityData, OnStartActivityData, OnSuccessActivityData } from "@microsoft/vscode-azext-utils/hostapi";
-import { Disposable, ThemeColor, ThemeIcon, TreeItemCollapsibleState } from "vscode";
-import { localize } from "../utils/localize";
+import { Disposable, ThemeColor, ThemeIcon, TreeItem, TreeItemCollapsibleState } from "vscode";
+import { ext } from "../../extensionVariables";
+import { localize } from "../../utils/localize";
+import { ResourceGroupsItem } from "../ResourceGroupsItem";
 
 export enum ActivityStatus {
     Running = 'running',
     Done = 'done'
 }
 
-export class ActivityTreeItem extends AzExtParentTreeItem implements Disposable {
-
+export class ActivityItem implements ResourceGroupsItem, Disposable {
```

Comment:
I think you can remove `hasMoreChildrenImpl` now since this is no longer an `AzExtParentTreeItem`